tests/experimental/datasources/test_pandas_filesystem_datasource.py

```diff
@@ -289,7 +289,7 @@ def test_data_asset_reader_options_passthrough(
         extra_kwargs: dict,
     ):
         batch_request = (
-            empty_data_context.sources.add_pandas_filesystem(
+            empty_data_context.sources.add_pandas_filesystem(  # type: ignore[attr-defined]
```

Comment:
These should go away once the stubs for all the `add_asset_methods` have been defined too.

Reply:
Oh, I believe this answers my question from above?